### PR TITLE
fix : Main Activity UI refreshed

### PR DESCRIPTION
**Pull request by dreadpool2**
First steps for issues #2446

Screenshots for the change:

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Bookmarks should be second last element and reduce the shadow of speakers and social links

Also, you removed the feature image, let it be there

**Comment by dreadpool2** [comment]
@iamareebjamal Okay :+1:

File: android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -344,4 +344,11 @@
 
     <string name="mobile_data_pref_warning">Wifi not found. Do you want to download using Mobile Data?</string>
 
+
+    <string name="summit_2018">Summit 2018</string>
```
Comment (nikit19) [inline android/app/src/main/res/values/strings.xml]: Is this string being used somewhere?

File: android/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -344,4 +344,11 @@
...
+    <string name="starts_on">Starts On</string>
+    <string name="ends_on">Ends On</string>
+    <string name="organizer_details">Organizer Details</string>
```
Comment (nikit19) [inline android/app/src/main/res/values/strings.xml]: I think this was Organiser Description earlier

**Review by nikit19 — CHANGES_REQUESTED** [review]
@dreadpool2 UI looks great !
I just saw these minor mistakes. Please have a look

File: android/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (dreadpool2) [reply android/app/src/main/res/values/strings.xml]: My bad ! Forgot to remove it ! Thanks for the review though :)

File: android/app/src/main/res/values/strings.xml (hunk 2 above)
Comment (dreadpool2) [reply android/app/src/main/res/values/strings.xml]: Okay :+1:

**Comment by dreadpool2** [comment]
@iamareebjamal Actually the Bookmarks section is present in the `fragment_about` file and in there we include the `content_about_event` layout where all other sections are present. Should I just migrate the Bookmarks section from the `fragment_about.xml` file ?

**Comment by dreadpool2** [comment]
@iamareebjamal Should I migrate the bookmarks section ?

**Comment by iamareebjamal** [comment]
Yes @dreadpool2

**Comment by dreadpool2** [comment]
@iamareebjamal Updated the PR :+1:

**Comment by dreadpool2** [comment]
@iamareebjamal Please review this

**Review by simarsingh24 — APPROVED** [review]

**Review by nikit19 — APPROVED** [review]

**Comment by iamareebjamal** [comment]
@dreadpool2 The logo is used twice whereas the background header is not used anywhere, Please fix this in follow up PR

**Review by iamareebjamal — APPROVED** [review]

**Comment by dreadpool2** [comment]
@iamareebjamal The logo is updated to header image once details are downloaded :+1:

**Pull request merged**